web/src/main/java/co/mobileaction/example/web/controller/PostController.java

```diff
@@ -41,4 +41,11 @@ public ResponseEntity<Boolean> deletePost(@PathVariable Long postId)
 
         return ResponseEntity.ok(true);
     }
+
+    @DeleteMapping("user/{userId}")

```

Comment:
```suggestion
    @DeleteMapping("/users/{userId}")

```  plural should be used in controller.

Reply:
it is said in the confluence sheet that the Client should send a request DELETE api/posts/user/{userId}. But i can change it if it is not necessary

Reply:
Common practice is to use nouns in plural and slash is better for readability and consistency.